CYCLONEDDS_QUALITY_DECLARATION.md

```diff
@@ -140,11 +180,16 @@ If `CycloneDDS` is built without security enabled, the product has no apparent r
 
 ## Platform Support [6]
 
-The `CycloneDDS` [repository documentation](https://github.com/eclipse-cyclonedds/cyclonedds/blob/master/README.md#building-eclipse-cyclone-dds) states that Linux, Mac and Windows 10 are supported.
-Platform version requirements are not explicitly stated, nor are all of the minimum build tool versions.
+`CycloneDDS` supports all of the tier 1 platforms as described in REP-2000, and tests each change against all of them.
+
+Currently nightly results can be seen here:
```

Comment:
Again these are test reports for the RMW layer. That implies Cyclone DDS supports the platform, but I don't think these exact links are the right place to point to.

Reply:
will reference cyclonedds, cyclonecdds-cxx CI travis testing & OR CI testing
https://travis-ci.com/github/eclipse-cyclonedds/cyclonedds